### PR TITLE
refactor: author profile cypress tests

### PR DESCRIPTION
**Pull request by L0wry**
enables the author profile tests in cypress

### CODE REVIEW

File: packages/fixture-generator/src/mock-author.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+class MockAuthor {
+  author: Author;
+
+  constructor() {
+    this.author = {
+      id: "1",
+      articles: getAuthorArticles(0),
+      biography: new MockMarkup().addParagraphs(1).get(),
+      image: "url",
+      jobTitle: "God",
```
Comment (parkjh) [inline packages/fixture-generator/src/mock-author.ts]: lol

File: packages/ssr/__tests__/integration/author-profile.js [hunk 2]
```diff
@@ -1,36 +1,42 @@
...
-    cy.url().should(
-      "eq",
-      "http://localhost:3000/article/97c64f20-cb67-11e4-a202-50ac5def393a"
-    );
+    expect(cy.get('[data-testid="standfirst"]')).to.exist;
   });
 
   it("loads inline-ad", () => {
-    cy.loadedAd("#inline-ad");
+    expect(cy.get("#inline-ad")).to.exist;
```
Comment (parkjh) [inline packages/ssr/__tests__/integration/author-profile.js]: was the command failing?

File: packages/ssr/__tests__/integration/author-profile.js [hunk 3]
```diff
@@ -1,36 +1,42 @@
...
+  );
+
   beforeEach(() => {
     cy.visit("/profile/fiona-hamilton");
   });
 
-  it("Author head has required elements", () => {
+  after(() => cy.task("stopMockServer"));
+
+  it("Should have the Author head required elements", () => {
```
Comment (shendriksen) [inline packages/ssr/__tests__/integration/author-profile.js]: Could rename the test: it("should have the required Author head elements")?

File: packages/ssr/__tests__/integration/author-profile.js [hunk 4]
```diff
@@ -1,36 +1,42 @@
...
     cy.get('div[data-testid="author-head"]');
     cy.get('h1[data-testid="author-name"]');
     cy.get('h2[role="heading"]');
     cy.get('div[data-testid="author-bio"]');
   });
 
-  it("Author has article elememts on the page", () => {
-    cy.get(`div[data-testid="${id}"]`);
-  });
+  it("Click on an article in the author article list takes you to the article page", () => {
```
Comment (shendriksen) [inline packages/ssr/__tests__/integration/author-profile.js]: Could rename this one: it("should take you to the article page once an article has been selected")

File: packages/ssr/__tests__/integration/author-profile.js [hunk 5]
```diff
@@ -1,36 +1,42 @@
...
-    );
+    expect(cy.get('[data-testid="standfirst"]')).to.exist;
   });
 
   it("loads inline-ad", () => {
-    cy.loadedAd("#inline-ad");
+    expect(cy.get("#inline-ad")).to.exist;
   });
 
   it("Next and Previous Pagination works", () => {
```
Comment (shendriksen) [inline packages/ssr/__tests__/integration/author-profile.js]: it("navigates between article pages")

File: packages/ssr/__tests__/integration/author-profile.js [hunk 6]
```diff
@@ -1,36 +1,42 @@
...
+  it("should take you to the article page once an article has been selected", () => {
+    cy.get(`div[data-testid="article-list-item-0"]`).click();
 
-  it("Click on an article in the author list takes you to the corresponding article", () => {
-    cy.get(`div[data-testid="${id}"]`).click();
-    cy.url().should(
-      "eq",
-      "http://localhost:3000/article/97c64f20-cb67-11e4-a202-50ac5def393a"
-    );
+    expect(cy.get('[data-testid="standfirst"]')).to.exist;
```
Comment (nehasri89) [inline packages/ssr/__tests__/integration/author-profile.js]: is there a reason we don't check for the URL?

File: packages/fixture-generator/src/mock-author.ts (hunk 1 above)
Comment (nehasri89) [inline packages/fixture-generator/src/mock-author.ts]: This is used at certain times in demo, can we have better fixtures pls?

Comment (L0wry) [reply packages/fixture-generator/src/mock-author.ts]: 😂 
sure thing

File: packages/ssr/__tests__/integration/author-profile.js (hunk 6 above)
Comment (L0wry) [reply packages/ssr/__tests__/integration/author-profile.js]: I'm not sure we should really be having hard-coded urls in the tests. There might be long-term plans to reuse these test on other environments 

 Also the ID that we check for in the url is now potentially not going to be static

Comment (nehasri89) [reply packages/ssr/__tests__/integration/author-profile.js]: 👍

**Pull request merged**